### ISSUE TITLE
How to generate sparql log from using LSQ?

### ISSUE THREAD

**Issue by Joeyzhouqihui** [dice-group/LSQ#39]
Hi, sorry for bothering you. I am working on a sparql system and I need some query log to test it. But I cannot find the command on the website about how to generate sparql log. Could you help me with it? Thank you so much!

**Comment by saleem-muhammad** [comment]
You cannot generate SPARQL query log using LSQ. Usually, such logs you get from public SPARQL endpoints. LSQ RDFize those logs.

**Comment by Joeyzhouqihui** [comment]
> 

But the "useful queries" on the website says we can "get all SELECT queries from DBpedia log" by executing "
SELECT DISTINCT ?text from <http://lsq.aksw.org/dbpedia> WHERE
{
   ?s <http://lsq.aksw.org/vocab#text> ?text .
   ?s <http://lsq.aksw.org/vocab#hasSpin> ?spin .
   ?spin a <http://spinrdf.org/sp#Select> . 
}
".
What does it mean? How can I execute that sparql using LSQ? Could you please give me some instructions? Thank you so much!

**Comment by saleem-muhammad** [comment]
Yes you can get SPARQL queries which were part of query logs from public SPARQL endpoints.  But the actual logs, we cannot provide due to providers policy.
